src/Generics/Deriving/TH/Internal.hs

```diff
@@ -515,7 +531,11 @@ elimTV _ptv ktv (KindedTV n k) = ktv n k
 -- This allows the library to be used in stage1 cross-compilers.
 
 gdPackageKey :: String
+#ifdef CURRENT_PACKAGE_KEY
 gdPackageKey = CURRENT_PACKAGE_KEY
+#else
+gdPackageKey = "generic-deriving-" ++ showVersion version
```

Comment:
this one is wrong. So this was removed on purpose.

Reply:
Thanks. I've pushed an update which leaves a comment so that this is easier to spot in the future.